tests/e2e/util/shell/shell.go

```diff
@@ -34,7 +34,7 @@ func ExecuteCommand(command string) (string, error) {
 
 // ExecuteShell executes a command given the input string and returns the output and err if any
 func ExecuteShell(command string, input string) (string, error) {
-	cmd := exec.Command("sh", "-c", command)
+	cmd := exec.Command("bash", "-c", command)
```

Comment:
Is this change needed? I'm not completely sure if this could break any of our test jobs. What if there's no bash on some of the systems we run these tests on? Do you really need to switch this to bash?